README.md

```diff
@@ -91,6 +91,11 @@ defaultTasks = ['contentDeploy']
 
 apply plugin: 'cognifide.aem'
 
+project.ext {
+  // where to put the bundle in the content package (optional)
+  bundlePath = "/apps/some/folder"
```

Comment:
instead of project ext property better if it will be value read from aemCompose.config.bundlePath